app.js

```diff
@@ -12,7 +13,62 @@ app.set('views', __dirname + '/views');
 app.use(express.static(__dirname + '/public'));
 
 // setting the spotify-api goes here:
+const spotifyApi = new SpotifyWebApi({
+    clientId: process.env.CLIENT_ID,
+    clientSecret: process.env.CLIENT_SECRET
+});
 
+spotifyApi
+    .clientCredentialsGrant()
+    .then(data => spotifyApi.setAccessToken(data.body['access_token']))
+    .catch(error => console.log('Something went wrong when retrieving an access token', error));
 // Our routes go here:
 
-app.listen(3000, () => console.log('My Spotify project running on port 3000 🎧 🥁 🎸 🔊'));
+
+app.get('/', (req, res, next) => {
+    res.render('index');
+})
+
+
+app.get('/artist-search/', (req, res, next) => {
+
+    const artist = req.query.artist
+
+    spotifyApi
+        .searchArtists(artist, { limit: 6 })
+        .then(data => {
+
+            res.render('artists-results', {
+                artists: data.body.artists.items,
+                data: data,
```

Comment:
You don't need to pass the data as you don't make use of it in the 'artists-results' hbs file.